src/pullCheckRunner.ts

```diff
@@ -9,7 +9,7 @@ export class PullCheckRunner {
     }
 
     public async rerunLastCheck() {
-        if (this.settings.payloadAction === "pull_request") {
+        if (this.settings.payloadAction === "pull_request_target") {
```

Comment:
This seems like a breaking change :thinking:  Though maybe that's OK.

Alternatively, we could handle both events as a possible trigger.